net/core/rtnetlink.c

```diff
@@ -2806,6 +2809,21 @@ static int do_setlink(const struct sk_buff *skb,
 				goto errout;
 			status |= DO_SETLINK_NOTIFY;
 		}
+
+#ifdef CONFIG_XDP_LUA
+		if (xdp[IFLA_XDP_LUA_PROG]) {
+			char *lua_prog = nla_data(xdp[IFLA_XDP_LUA_PROG]);
```

Comment:
script? I think we should uniformize it.. we are using both `lua_prog` and `lua_script` in the code.. I would rename them all to `script`